package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "typeshed",
+    "version": "0.0.0",
```

Comment:
These fields are required in order for this to be considered a valid `package.json` file: https://docs.npmjs.com/creating-a-package-json-file#packagejson-fields

Reply:
Just FYI if you ever want to add a comment to a package.json file, just add any top level field that starts with `//`. ie:
`"//": "my comment"`, `"// some-packge": "other comment about a package, with a unique comment key"` (the official recommendation is to make `"//"` as array, but that lacks comment positioning, so I've been doing it the way I described for years)
https://groups.google.com/g/nodejs/c/NmL7jdeuw0M/m/yTqI05DRQrIJ